contracts/utils/FlashLiquidator.sol

```diff
@@ -102,7 +103,7 @@ contract FlashLiquidator is IEulerFlashLoanReceiver, BoringOwnable {
             
             require(maxLiquidatorDepositAssetCash > 0);
 
-            NOTIONAL.deleverageAccount(
+            IStrategyVault(params.vault).deleverageAccount(
```

Comment:
This might need to be regular Notional sometimes if there is no reentrancy guard check.

We should add a param to decide which contract it will call here

Reply:
Fixed